tensorboard/examples/plugins/example_angular/tensorboard_plugin_angular/frontend/.gitignore

```diff
@@ -0,0 +1,46 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+
+# compiled output
+/dist
+/tmp
+/out-tsc
+# Only exists if Bazel was run
+/bazel-out
```

Comment:
few questions: is this file strictly required? can we merge it with example_angular/.gitignore? Can some of these be included to .gitignore?

Reply:
Done in #2649